### PR TITLE
[CF-376] Use boot_timeout and storage_backend_timeout options

### PR DESCRIPTION
**Pull request by SVilgelm**
Use value of timeout options in any calls of wait_for_status function.

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module tests.test
tests/test.py:21: [E0611(no-name-in-module), ] No name 'cloud_ferry' in module 
'tests.cloud'
tests/test.py:39: [E1101(no-member), TestCase.cloud_config] Module 'tests.cloud' has no 
'Cloud' member
************* Module tests.cloudferrylib.os.compute.test_nova
tests/cloudferrylib/os/compute/test_nova.py:450: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:453: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:455: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:483: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:484: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:488: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:490: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:518: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:519: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:523: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:525: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _add_flavor_access_for_tenants of a client class

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check failed
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/3045/

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests failed
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/3050/

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module tests.test
tests/test.py:21: [E0611(no-name-in-module), ] No name 'cloud' in module 'tests.cloud'
tests/test.py:21: [W0403(relative-import), ] Relative import 'cloud', should be 
'tests.cloud'
************* Module tests.cloudferrylib.os.compute.test_nova
tests/cloudferrylib/os/compute/test_nova.py:450: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:453: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:455: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:483: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:484: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:488: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:490: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:518: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:519: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:523: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:525: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _add_flavor_access_for_tenants of a client class

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by yosharma** [comment]
+1
Tested on staging environment

File: cloudferrylib/os/actions/detach_used_volumes_via_compute.py [hunk 1]
```diff
@@ -28,19 +31,20 @@ def run(self, info, **kwargs):
         compute_resource = self.cloud.resources[utils.COMPUTE_RESOURCE]
         storage_resource = self.cloud.resources[utils.STORAGE_RESOURCE]
         for instance in info[utils.INSTANCES_TYPE].itervalues():
-            LOG.debug("Detaching volumes for instance %s [%s]" %
-                      (instance['instance']['name'],
-                       instance['instance']['id']))
+            LOG.debug("Detaching volumes for instance %s [%s]",
```
Comment (roman-verchikov) [inline cloudferrylib/os/actions/detach_used_volumes_via_compute.py]: `LOG.info`

File: cloudferrylib/os/actions/live_migration.py [hunk 2]
```diff
@@ -176,7 +178,7 @@ class RestoreOriginalInstancesTableSchema(action.Action):
     Requirements:
       - Write access to nova DB on destination
     """
-    def run(self, info=None, **kwargs):
+    def run(self, info=None, **kwargs):  # pylint: disable=unused-argument
```
Comment (roman-verchikov) [inline cloudferrylib/os/actions/live_migration.py]: Let's keep the pylint warning

File: cloudferrylib/os/actions/live_migration.py [hunk 3]
```diff
@@ -195,7 +197,7 @@ class CheckQemuVersion(action.Action):
 
     REQUIRED_QEMU_VERSION = "2.0"
 
-    def run(self, info=None, **kwargs):
+    def run(self, info=None, **kwargs):  # pylint: disable=unused-argument
```
Comment (roman-verchikov) [inline cloudferrylib/os/actions/live_migration.py]: Let's keep the pylint warning

File: cloudferrylib/os/compute/nova_compute.py [hunk 4]
```diff
@@ -857,10 +857,11 @@ def change_status(self, status, instance=None, instance_id=None):
             'unpaused': lambda instance: instance.unpause(),
             'suspended': lambda instance: instance.suspend(),
             'confirm_resize': lambda instance: instance.confirm_resize(),
+            # pylint: disable=undefined-variable
```
Comment (roman-verchikov) [inline cloudferrylib/os/compute/nova_compute.py]: Let's keep the warning, it reminds that this needs to be refactored

File: tests/cloudferrylib/os/compute/test_nova.py [hunk 5]
```diff
@@ -13,6 +13,7 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
+# pylint: disable=protected-access
```
Comment (roman-verchikov) [inline tests/cloudferrylib/os/compute/test_nova.py]: Let's keep pylint warning, it reminds that tests are written incorrectly and require refactoring

**Comment by roman-verchikov** [comment]
-1, please resolve inlines

File: cloudferrylib/os/actions/detach_used_volumes_via_compute.py (hunk 1 above)
Comment (SVilgelm) [inline cloudferrylib/os/actions/detach_used_volumes_via_compute.py]: done

File: cloudferrylib/os/actions/live_migration.py (hunk 2 above)
Comment (SVilgelm) [inline cloudferrylib/os/actions/live_migration.py]: I've removed `info` variable, should work

File: cloudferrylib/os/actions/live_migration.py (hunk 3 above)
Comment (SVilgelm) [inline cloudferrylib/os/actions/live_migration.py]: ditto

File: cloudferrylib/os/compute/nova_compute.py (hunk 4 above)
Comment (SVilgelm) [inline cloudferrylib/os/compute/nova_compute.py]: Done

File: tests/cloudferrylib/os/compute/test_nova.py (hunk 5 above)
Comment (SVilgelm) [inline tests/cloudferrylib/os/compute/test_nova.py]: Done

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferrylib.os.compute.nova_compute
cloudferrylib/os/compute/nova_compute.py:863: [E0602(undefined-variable), 
NovaCompute.change_status.<lambda>.<lambda>] Undefined variable 'status'
************* Module tests.cloudferrylib.os.compute.test_nova
tests/cloudferrylib/os/compute/test_nova.py:449: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:452: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:454: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:482: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:483: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:487: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:489: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:517: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:518: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:522: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:524: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _add_flavor_access_for_tenants of a client class

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferrylib.os.actions.attach_used_volumes_via_compute
cloudferrylib/os/actions/attach_used_volumes_via_compute.py:18: [E0401(import-error), ] 
Unable to import 'cinderclient'
cloudferrylib/os/actions/attach_used_volumes_via_compute.py:19: [E0401(import-error), ] 
Unable to import 'novaclient'
cloudferrylib/os/actions/attach_used_volumes_via_compute.py:20: [E0401(import-error), ] 
Unable to import 'oslo_config'
************* Module cloudferrylib.os.actions.detach_used_volumes_via_compute
cloudferrylib/os/actions/detach_used_volumes_via_compute.py:18: [E0401(import-error), ] 
Unable to import 'oslo_config'
************* Module cloudferrylib.os.actions.convert_volume_to_image
cloudferrylib/os/actions/convert_volume_to_image.py:18: [E0401(import-error), ] Unable 
to import 'oslo_config'
************* Module cloudferrylib.os.actions.check_cloud
cloudferrylib/os/actions/check_cloud.py:20: [E0401(import-error), ] Unable to import 
'cinderclient'
cloudferrylib/os/actions/check_cloud.py:21: [E0401(import-error), ] Unable to import 
'glanceclient'
cloudferrylib/os/actions/check_cloud.py:22: [E0401(import-error), ] Unable to import 
'keystoneclient.openstack.common.apiclient'
cloudferrylib/os/actions/check_cloud.py:23: [E0401(import-error), ] Unable to import 
'neutronclient.common'
cloudferrylib/os/actions/check_cloud.py:24: [E0401(import-error), ] Unable to import 
'novaclient'
cloudferrylib/os/actions/check_cloud.py:25: [E0401(import-error), ] Unable to import 
'oslo_config'
************* Module cloudferrylib.os.actions.live_migration
cloudferrylib/os/actions/live_migration.py:44: [E0401(import-error), ] Unable to import 
'fabric.operations'
cloudferrylib/os/actions/live_migration.py:45: [E0401(import-error), ] Unable to import 
'oslo_config'
************* Module cloudferrylib.os.storage.cinder_storage
cloudferrylib/os/storage/cinder_storage.py:18: [E0401(import-error), ] Unable to import 
'cinderclient.v1'
cloudferrylib/os/storage/cinder_storage.py:19: [E0401(import-error), ] Unable to import 
'cinderclient'
************* Module tests.test
tests/test.py:16: [E0401(import-error), ] Unable to import 'mock'
tests/test.py:18: [E0401(import-error), ] Unable to import 'oslotest'
************* Module tests.cloudferrylib.os.storage.test_cinder_storage
tests/cloudferrylib/os/storage/test_cinder_storage.py:16: [E0401(import-error), ] 
Unable to import 'mock'
tests/cloudferrylib/os/storage/test_cinder_storage.py:18: [E0401(import-error), ] 
Unable to import 'cinderclient.v1'
tests/cloudferrylib/os/storage/test_cinder_storage.py:19: [E0401(import-error), ] 
Unable to import 'oslotest'
************* Module tests.test_cloud.test_grouping
tests/test_cloud/test_grouping.py:19: [E0401(import-error), ] Unable to import 'mock'
tests/test_cloud/test_grouping.py:21: [E0401(import-error), ] Unable to import 
'oslotest'
************* Module tests.cloudferrylib.os.compute.test_nova
tests/cloudferrylib/os/compute/test_nova.py:16: [E0401(import-error), ] Unable to 
import 'mock'
tests/cloudferrylib/os/compute/test_nova.py:18: [E0401(import-error), ] Unable to 
import 'novaclient.v1_1'
tests/cloudferrylib/os/compute/test_nova.py:19: [E0401(import-error), ] Unable to 
import 'oslotest'
tests/cloudferrylib/os/compute/test_nova.py:449: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:452: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:454: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:482: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:483: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:487: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:489: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:517: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:518: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:522: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:524: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _add_flavor_access_for_tenants of a client class
************* Module cloudferrylib.os.compute.nova_compute
cloudferrylib/os/compute/nova_compute.py:22: [E0401(import-error), ] Unable to import 
'novaclient.v1_1'
cloudferrylib/os/compute/nova_compute.py:23: [E0401(import-error), ] Unable to import 
'novaclient'
cloudferrylib/os/compute/nova_compute.py:863: [E0602(undefined-variable), 
NovaCompute.change_status.<lambda>.<lambda>] Undefined variable 'status'

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferrylib.os.actions.attach_used_volumes_via_compute
cloudferrylib/os/actions/attach_used_volumes_via_compute.py:18: [E0401(import-error), ] 
Unable to import 'cinderclient'
cloudferrylib/os/actions/attach_used_volumes_via_compute.py:19: [E0401(import-error), ] 
Unable to import 'novaclient'
cloudferrylib/os/actions/attach_used_volumes_via_compute.py:20: [E0401(import-error), ] 
Unable to import 'oslo_config'
************* Module cloudferrylib.os.actions.detach_used_volumes_via_compute
cloudferrylib/os/actions/detach_used_volumes_via_compute.py:18: [E0401(import-error), ] 
Unable to import 'oslo_config'
************* Module cloudferrylib.os.actions.convert_volume_to_image
cloudferrylib/os/actions/convert_volume_to_image.py:18: [E0401(import-error), ] Unable 
to import 'oslo_config'
************* Module cloudferrylib.os.actions.live_migration
cloudferrylib/os/actions/live_migration.py:44: [E0401(import-error), ] Unable to import 
'fabric.operations'
cloudferrylib/os/actions/live_migration.py:45: [E0401(import-error), ] Unable to import 
'oslo_config'
************* Module cloudferrylib.os.actions.check_cloud
cloudferrylib/os/actions/check_cloud.py:20: [E0401(import-error), ] Unable to import 
'cinderclient'
cloudferrylib/os/actions/check_cloud.py:21: [E0401(import-error), ] Unable to import 
'glanceclient'
cloudferrylib/os/actions/check_cloud.py:22: [E0401(import-error), ] Unable to import 
'keystoneclient.openstack.common.apiclient'
cloudferrylib/os/actions/check_cloud.py:23: [E0401(import-error), ] Unable to import 
'neutronclient.common'
cloudferrylib/os/actions/check_cloud.py:24: [E0401(import-error), ] Unable to import 
'novaclient'
cloudferrylib/os/actions/check_cloud.py:25: [E0401(import-error), ] Unable to import 
'oslo_config'
************* Module cloudferrylib.os.storage.cinder_storage
cloudferrylib/os/storage/cinder_storage.py:18: [E0401(import-error), ] Unable to import 
'cinderclient.v1'
cloudferrylib/os/storage/cinder_storage.py:19: [E0401(import-error), ] Unable to import 
'cinderclient'
************* Module cloudferrylib.os.compute.nova_compute
cloudferrylib/os/compute/nova_compute.py:22: [E0401(import-error), ] Unable to import 
'novaclient.v1_1'
cloudferrylib/os/compute/nova_compute.py:23: [E0401(import-error), ] Unable to import 
'novaclient'
cloudferrylib/os/compute/nova_compute.py:863: [E0602(undefined-variable), 
NovaCompute.change_status.<lambda>.<lambda>] Undefined variable 'status'
************* Module tests.cloudferrylib.os.storage.test_cinder_storage
tests/cloudferrylib/os/storage/test_cinder_storage.py:18: [E0401(import-error), ] 
Unable to import 'cinderclient.v1'
************* Module tests.cloudferrylib.os.compute.test_nova
tests/cloudferrylib/os/compute/test_nova.py:18: [E0401(import-error), ] Unable to 
import 'novaclient.v1_1'
tests/cloudferrylib/os/compute/test_nova.py:449: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:452: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:454: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:482: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:483: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:487: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:489: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:517: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:518: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:522: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:524: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _add_flavor_access_for_tenants of a client class

**Comment by SVilgelm** [comment]
retest this please

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferrylib.os.compute.nova_compute
cloudferrylib/os/compute/nova_compute.py:863: [E0602(undefined-variable), 
NovaCompute.change_status.<lambda>.<lambda>] Undefined variable 'status'
************* Module tests.cloudferrylib.os.compute.test_nova
tests/cloudferrylib/os/compute/test_nova.py:449: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:452: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:454: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:482: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:483: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:487: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:489: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:517: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:518: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:522: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:524: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _add_flavor_access_for_tenants of a client class

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by SVilgelm** [comment]
retest this please

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferrylib.os.compute.nova_compute
cloudferrylib/os/compute/nova_compute.py:863: [E0602(undefined-variable), 
NovaCompute.change_status.<lambda>.<lambda>] Undefined variable 'status'
************* Module tests.cloudferrylib.os.compute.test_nova
tests/cloudferrylib/os/compute/test_nova.py:449: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:452: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:454: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:482: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:483: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:487: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:489: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:517: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:518: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:522: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:524: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _add_flavor_access_for_tenants of a client class

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by yosharma** [comment]
+1, tested in staging environment.Working fine.

**Comment by SVilgelm** [comment]
retest this please

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferrylib.os.compute.nova_compute
cloudferrylib/os/compute/nova_compute.py:863: [E0602(undefined-variable), 
NovaCompute.change_status.<lambda>.<lambda>] Undefined variable 'status'
************* Module tests.cloudferrylib.os.compute.test_nova
tests/cloudferrylib/os/compute/test_nova.py:449: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:452: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:454: [W0212(protected-access), 
FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access 
to a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:482: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:483: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:487: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:489: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a 
protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:517: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:518: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:522: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:524: [W0212(protected-access), 
FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to 
a protected member _add_flavor_access_for_tenants of a client class

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by roman-verchikov** [comment]
+1, can be merged

**Pull request merged**